client/src/pages/Login.tsx

```diff
@@ -0,0 +1,130 @@
+import React, { useEffect, useState } from "react";
+import { fetchJson, fetchText } from "./../utils";
+import { TextInputField } from "./../components/Components";
+import { Container, Grid, Typography, Button } from "@material-ui/core";
+import { IdentityToken, emptyIdentityToken } from "./../interfaces";
+import Logo from "./../img/Logo.png";
+import "./../css/Login.css";
+
+// Applease typescript
+declare const window: any;
+// Grab handle to our microapps js library
+const microapps = window.microapps;
+
+function Login() {
+  const is_web = window.location == window.parent.location;
```

Comment:
Add a comment to explain why the is_web can be determined by comparing window.location and window.parent.location

Reply:
variable moved to constants file and updated comments